templates/create-daml-app-test-resources/index.test.ts

```diff
@@ -178,6 +179,7 @@ const newUiPage = async (): Promise<Page> => {
     throw Error('Puppeteer browser has not been launched');
   }
   const page = await browser.newPage();
+  await page.setViewport({ width: 1366, height: 1080});
```

Comment:
Do we need this to handle overflows of party ids or something like that?

Reply:
and if so, do we still need it if we add aliases for messages?

Reply:
it looks like mac/linux behave slightly different when rendering the party ids. this fixes it. I'll change the aliases for messages in the next PR, but I think it still makes sense to set the viewport explicitly.